src/components/KFileUpload/KFileUpload.vue

```diff
@@ -64,14 +68,19 @@
   </div>
 </template>
 
+<script lang="ts">
+export default {
+  inheritAttrs: false,
+}
+</script>
+
```

Comment:
nit: we can add `defineOptions` in the `<script lang="ts" setup>` block:
```ts
defineOptions({
  inheritAttrs: false,
})
```

Then we don't need another `<script>` block.

Reply:
I’d actually recommend making this change anywhere we’re already using inheritors: false

Reply:
That produces typescript errors

![Screenshot 2024-06-18 at 10 51 57 PM](https://github.com/Kong/kongponents/assets/36751160/65a8dc11-bd53-41ea-a995-1690f4dc637e)

Reply:
Oh, sorry, you suggest using `defineOptions`. Pushed a fix

Reply:
I can handle updating that in every component other than ones scoped in this PR in a separate PR